src/services/codefixes/fixAddMissingPropertyInitializer.ts

```diff
@@ -0,0 +1,112 @@
+/* @internal */
+namespace ts.codefix {

```

Comment:
you do not need to split them into 3 fixes, one code fix can return multiple actions. for instance see https://github.com/Microsoft/TypeScript/blob/master/src/services/codefixes/fixUnusedIdentifier.ts#L11 we will add two actions possibly, one for `_` prefix, and one for deleting the declaration, and the user gets to choose.